packages/ListBoxBrowser/README.md

```diff
@@ -22,90 +22,54 @@ npm install @paprika/listbox-browser
+| height                 | number | false    | 220      | Set the height for the ListBoxBrowser                                                                                                                                                                                                                     |
+| onChange               | func   | false    | () => {} | Callback function receiving an array of selected options by the component                                                                                                                                                                                 |
+| isParentSelectable     | bool   | false    | null     | Allows the user to select the parent options                                                                                                                                                                                                              |
+| rootTitle              | node   | false    | ""       | Content title for the left column                                                                                                                                                                                                                         |
+| browserTitle           | node   | false    | ""       | Content title for the right column                                                                                                                                                                                                                        |
+| children               | node   | false    | null     | You can pass <ListBoxBrowser.OptionSelected /> as a children                                                                                                                                                                                              |
+| hasBreadcrumb          | bool   | false    | true     | Indicates if the right column should display a breadcrumb                                                                                                                                                                                                 |
+| hasError               | bool   | false    | false    | Set a border red color around the component indicating that has an error                                                                                                                                                                                  |
+| onFetch                | func   | false    | null     | When declaring the array options empty, this will be executed to retrieve the data, useful if you want to do a lazy load.                                                                                                                                 |
+| defaultSelectedOptions | func   | false    | () => {  |
```

Comment:
This props table is broken because of the **defaultSelectedOptions** default value. I think the issue is, its multi-line. I tried making it one line but prettier keeps on updating it.

Reply:
will fix once we merge it into my PR